about.html

```diff
@@ -0,0 +1,292 @@
+								the world of work.
+							</p>
+						</div>
+
+						<div class=" p-4 text-center text-dark shadow mb-2" >
+							<p>
+								Please contact us per Email for nay further questions about
+								VICTORIA UNIVERSITY ALUMNI STUDENT SUMMIT 2023!
+							</p>
+							<p>admissions@vu.ac.ug /marketing@vu.ac.ug</p>
+						</div>
+					</div>
+
+					<div class=" py-2 ">
+						<h1 class="h1 text-danger text-center mb-3 shadow p-4 mb-2 ">
+							VU ALUMNI STUDENT SUMMIT 2023 CREATIVE LOGO 
+						</h1>
+						<div class="d-flex justify-content-center"><hr class="bar bg-danger"> </hr></div>
+					</div>
+
+					<div class="bgColor-2 p-3">
+						<p class="hero-intro text-center border border-5 border-white textColor h6">
+							The logo of Alumni VU Summit 2023 was taking from the university official logo. </p>
+							<p class="hero-intro text-center border border-5 border-white textColor" id="hero-into-caption">Our current students have many opportunities to meet our alumni in both social and 
+								professional settings where personal and career experiences are shared.</p>
+						</p>
+							
+					</div>
+					<!-- logo -->
+					
+						<div class="h1 text-danger text-center mb-3 shadow p-4 mb-2 border border-white" id="vu-summit-logo">
+							<img
+							class="w-25 h-25"
+							src="./asset/images/victoria-university-logo.png"
+							alt="Victoria-university-logo" /> <br>
+							<small class="text-small">26th~ 28th Agu 2023 Victoria university,Kampala uganda</small>
+						</div>
+				</section>
+<!-- previous summit -->
+				<section>
+					<div class="shadow p-4 mb-2">
+						<h3 class="h3 text-center ">
+							See the past VU Alumni Summits
+						</h3>
+						<div class="d-flex justify-content-center"><hr class="bar bg-danger"> </hr></div>
+						<p class="hero-intro text-center h6">
+							Take a look at the past VU Alumni Summits which took place in Kampala
+						</p>
+						
+					</div>
+					<div class="past-summits">
+						<div class="bgConference-img-2019  m-2 shadow px-2 ">
+							<h5 class=" display-4 text-light text-center mt-2 p-5" id="conference-title">2016 VU Alumni Summits </h2>
+						</div>
+	
+						<div class="bgConference-img-2019 shadow m-2 px-2 ">
+							<h5 class="display-4 text-light text-center mt-2 p-5 " id="conference-title"> 2019 VU Alumni Summits </h2>
+						</div>
+
+					</div>
+					
+				</section>
+			</main>
+			<footer>
+				<div class="mobile-footer-hidden">
+				<div class="bgColor3   text-center text-light shadow mb-2" >
+					<h3 class="display-6 text-light text-center ">Partners</h3>
+
+					<ul class="partner d-flex justify-content-between align-items-center">
+						<li class="mx-4"><img class="partner-img" src="./asset/images/ruparelia.jpg" alt=""></li>
+						<li class="mx-4"><img class="partner-img"src="./asset/images/Speke-Resort-logo.png" alt=""></li>
+					</ul>
+				</div>
+
+				<div class=" text-center shadow p-2 mb-2 d-flex justify-content-between">
+					<h6 >
```

Comment:
[REQUIRED] Kindly change the footer's background in the desktop screen size to black colour